### ISSUE TITLE
Error Installing Artifactory Extension on TFS 2018

### ISSUE THREAD

**Issue by troyattaxact** [jfrog/artifactory-vsts-extension#73]

**Comment by troyattaxact** [comment]
I've been unsuccessful installing the Artifactory extension on TFS 2018. @MightyJay recommended using version 1.3.1 of the extension but I'm unable to locate that version. Installing version 1.5.0 gives me the following error:
Following errors were found in artifact contribution:defaultVersionType is not supported for Artifact source contribution.

**Comment by eyalbe4** [comment]
@troyattaxact,
A college of mine managed to fix this issue by upgrading TFS.
He said he'll soon share some more details here.

**Comment by amithkm** [comment]
@troyattaxact,
The error "Following errors were found in artifact contribution:defaultVersionType is not supported for Artifact source contribution." is gone when I installed tfsserver2018.3.2.exe. Can you please try to install the mentioned version and let us know if you are able to resolve it. The version it showed after installing from the TFS UI is 16.131.28601.4

**Comment by MightyJay** [comment]
I just tried it again using the method I described in issue #66, but now I also only get version 1.5.0 and it no longer shows me the 1.3.1 version. I can't see a method to select an older version, so I guess I was just lucky that I got it to install last time. I can only see the older version when I browse the extension gallery in my local TFS server (since I have already installed that 1.3.1 extension on my TFS server).

If you have the option to update TFS like @amithkm suggests (perhaps even to the new Azure DevOps server), that should fix your problem. Otherwise I don't know how to get an older version of this plugin installed.
